### PR TITLE
add return values for words and word_ids; add tests

### PR DESCRIPTION
**Pull request by kyleclo**

### CODE REVIEW

File: smashed/mappers/tokenize.py [hunk 1]
```diff
@@ -59,16 +61,37 @@ def __init__(
         if return_length:
             output_fields.append(f'{self.prefix}length')
 
+        if 'is_split_into_words' in tk_kwargs and return_word_ids:
```
Comment (soldni) [inline smashed/mappers/tokenize.py:64]: could we make this an explicit argument of the `__init__` method, and then add it to `tk_kwargs` in lines 30-32?

File: smashed/mappers/tokenize.py [hunk 2]
```diff
@@ -59,16 +61,37 @@ def __init__(
...
         self.input_fields = input_fields
         self.output_fields = output_fields
 
     def transform(self, data: TransformElementType) -> TransformElementType:
         batch_encoding = self.tokenizer(data[self.input_fields[0]],
                                         **self.tokenize_kwargs)
-        batch_encoding = {f'{self.prefix}{k}': v
-                          for k, v in batch_encoding.items()}
-        return dict(batch_encoding)
+        batch_encoding_as_dict = {f'{self.prefix}{k}': v
```
Comment (soldni) [inline smashed/mappers/tokenize.py:76]: I would do this as last step: use `BatchEncoding` object as normal, but the, before returning, call a function that adds the prefix + casts

**Review by soldni — CHANGES_REQUESTED** [review]

File: smashed/mappers/tokenize.py (hunk 1 above)
Comment (kyleclo) [reply smashed/mappers/tokenize.py:64]: done

File: smashed/mappers/tokenize.py (hunk 2 above)
Comment (kyleclo) [reply smashed/mappers/tokenize.py]: done

**Review by soldni — APPROVED** [review]

**Pull request merged**